schema.js

```diff
@@ -12,6 +12,7 @@ module.exports = Joi.object().keys({
       dataHost: Joi.string(),
       s3Options: Joi.string(),
       adminLookup: Joi.boolean(),
+      token: Joi.string(),
```

Comment:
This change is actually unrelated but should exist since https://github.com/pelias/config/pull/137